terraform/southpark.tf

```diff
@@ -0,0 +1,170 @@
+provider "aws" {
+  version = "~> 2.0"
+  region  = "us-east-2"
+}
+
+
+#Create repo
+resource "aws_ecr_repository" "southpark_repo" {
```

Comment:
what is exactly a `repo` in this context?

Reply:
And AWS repository

Reply:
It is a docker image repository, where you can push/pull docker images. It is the default way in AWS to use docker images.

Anyhow, we could also use the docker registry, where we usually put our docker images (is it docker-hub?) as described in here: https://aws.amazon.com/blogs/devops/how-to-use-docker-images-from-a-private-registry-in-aws-codebuild-for-your-build-environment/

Reply:
Thanks Max!